protocol/messenger.go

```diff
@@ -2218,13 +2218,18 @@ func (m *Messenger) sendChatMessage(ctx context.Context, message *common.Message
 
 	message.DisplayName = displayName
 	if len(message.ImagePath) != 0 {
-
 		err := message.LoadImage()
 		if err != nil {
 			return nil, err
 		}
+	} else if len(message.AudioPath) != 0 {
+		err := message.LoadAudio()
+		if err != nil {
+			return nil, err
+		}
+	}
 
-	} else if len(message.CommunityID) != 0 {
+	if len(message.CommunityID) != 0 {
```

Comment:
I think this code path is only to send an invitation to the community. So it's probably not the culprit

Reply:
I tried to split the logic here a bit, so it's more obvious what we're checking for. Will keep this bit.